<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Adskii_Rasgon" failures="0" tests="5" name="test.java.lesson14_testng.FactorialTest" time="0.017" errors="0" timestamp="2024-10-20T19:18:43 UZT" skipped="0">
  <testcase classname="test.java.lesson14_testng.FactorialTest" name="testFactorialThrowsExceptionForNegativeNumber" time="0.001"/>
  <system-out/>
  <testcase classname="test.java.lesson14_testng.FactorialTest" name="testFactorial" time="0.012"/>
  <system-out/>
  <testcase classname="test.java.lesson14_testng.FactorialTest" name="testFactorial" time="0.001"/>
  <system-out/>
  <testcase classname="test.java.lesson14_testng.FactorialTest" name="testFactorial" time="0.001"/>
  <system-out/>
  <testcase classname="test.java.lesson14_testng.FactorialTest" name="testFactorial" time="0.002"/>
  <system-out/>
</testsuite> <!-- test.java.lesson14_testng.FactorialTest -->
